src/Microsoft.AspNet.Mvc.Core/ActionResults/ContentResult.cs

```diff
@@ -11,11 +13,17 @@ namespace Microsoft.AspNet.Mvc
 {
     public class ContentResult : ActionResult
     {
-        public string Content { get; set; }
+        private readonly MediaTypeHeaderValue DefaultContentType = new MediaTypeHeaderValue("text/plain")
+        {
+            Encoding = Encoding.UTF8
+        };
 
-        public Encoding ContentEncoding { get; set; }
```

Comment:
yeah, right...(as a side note...this property name is just confusing...as per Http this would represent the compression encoding like gzip, deflate)..